data/templates/legacy-article.mst

```diff
@@ -11,13 +11,12 @@
     {{/title}}
     <section id="inside-content" class="section">
         <div>{{{body-html}}}</div>
-        <hr>
```

Comment:
Good point, but as far as I know, `<hr>` and borders are different. 
In the .disclaimer class, I added a top-border, which adds a border to the .disclaimer div --- it's not manipulating the `<hr>`. They just look very similar.